### PR TITLE
Add HW6

### PR DESCRIPTION
**Pull request by DariaBoris**

### CODE REVIEW

File: hw6/Advanced_Task_2.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    name: 'Polina',
+    age: 27
+}
+
+let student2 = {
+    name: 'Polina',
+    age: 27
+}
+
+function compareObj(student1: any, student2: any): any {
```
Comment (HeorhSemianenia) [inline hw6/Advanced_Task_2.ts]: return type shouldn't be any

Comment (DariaBoris) [reply hw6/Advanced_Task_2.ts]: Fixed.

**Comment by HeorhSemianenia** [comment]
image

File: hw6/Advanced_Task_2.ts (hunk 1 above)
Comment (HeorhSemianenia) [reply hw6/Advanced_Task_2.ts]: Not really
"Сравните их так чтобы они были равны" but what result do you currently have?

Comment (DariaBoris) [reply hw6/Advanced_Task_2.ts]: Fixed.

**Comment by DariaBoris** [comment]
Created a new branch, hope it helps with deleting unnecessary files.

**Pull request closed without merging**